website/src/types/Hooks.ts

```diff
@@ -1,16 +1,11 @@
-import { BaseTask, TaskContent, TaskResponse, TaskType } from "src/types/Task";
+import { BaseTask, TaskResponse, TaskType } from "src/types/Task";
 
-interface TaskInteraction {
-  id: string;
-  update_type: string;
-  content: TaskContent;
-}
-
-export type TaskApiHook<Task extends BaseTask> = {
+export type TaskApiHook<Task extends BaseTask, ResponseContent> = {
   response: TaskResponse<Task>;
   isLoading: boolean;
-  completeTask: (interaction: TaskInteraction) => void;
-  skipTask: () => void;
+  completeTask: (interaction: ResponseContent) => Promise<void>;
+  skipTask: () => Promise<void>;
```

Comment:
I don't think we need skipTask if we have rejectTask. Probably best not to expose a way to accidentally skip without informing the server we are rejecting.

Reply:
Agree, I will remove it in the next PR.